README.md

```diff
@@ -107,18 +118,18 @@ Example:
     # Grant access only to paths under /api
     --openshift-delegate-urls='{"/api":{"group":"custom.group","resource":"myproxy""verb":"get"}}'
 
-WARNING: Because users are sending their own credentials to the proxy, it's important to use this 
-setting only when the proxy is under control of the cluster administrators. Otherwise, end users 
+WARNING: Because users are sending their own credentials to the proxy, it's important to use this
+setting only when the proxy is under control of the cluster administrators. Otherwise, end users
 may be unwittingly provide their credentials to untrusted components that can then act as them.
 
 
 ### Other configuration flags
 
 #### `--openshift-service-account=NAME`
 
-Will attempt to read the `--client-id` and `--client-secret` from the service account information 
+Will attempt to read the `--client-id` and `--client-secret` from the service account information
 injected by OpenShift. Uses the value of `/var/run/secrets/kubernetes.io/serviceaccount/namespace`
```

Comment:
nit: can you avoid touching unrelated code/documentation lines ?

Reply:
Sure thing, just my editor removing extra spaces. Done.